.migration-lint-ignore.json

```diff
@@ -0,0 +1,22 @@
+{
+  "procedures": [
+    "accountEmails_4",
```

Comment:
Would be nice to know what lint rules a procedure/table is breaking...but meh.

Reply:
Tbh, I see the value of these lints not in checking migrations that we already wrote, but in checking migrations we're yet to write.

If we're really interested in why something fails right now, we can just remove the entry here and run the script. It will tell us, but that knowledge is academic because we can't go back and change the old migration anyway. But when write a new migration, it will tell us what we've done wrong at that point, which is where the real value is. Hopefully that will stop us deploying problematic migrations to prod in the future.